### PR TITLE
Add 20230427 ops team minutes

### PR DESCRIPTION
**Pull request by joshmue**

### CODE REVIEW

**Comment by joshmue** [comment]
@fkr feel free to add yourself to the attendees, if you want.

**Comment by fkr** [comment]
@joshmue ooops. will do!

File: ops/20230427.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# 2023-04-27
+## Participants  use of github handle is preferred
+- @joshmue
+- @garloff
+- @o-otte
+- @matfechner
+- @JoshuaI96
+- @frosty-geek
+
```
Comment (fkr) [inline ops/20230427.md:9]:
```suggestion
- @fkr 
```

**Review by fkr — APPROVED** [review]
LGTM! Thanks @joshmue.

**Pull request merged**